packages/rspack/tests/cases/context/issue-3014/index.js

```diff
@@ -0,0 +1,4 @@
+it("should automatically infer the index.js file", function () {
```

Comment:
Does this description meaningful? Or you could just write `it("issue-3014")`.

Reply:
fixed